scripts/nb-tester/test-notebook.py

```diff
@@ -30,18 +30,21 @@
     "docs/api/**",
     "**/.ipynb_checkpoints/**",
 ]
-NOTEBOOKS_THAT_SUBMIT_JOBS = [
-    "docs/start/hello-world.ipynb",
-    "docs/analyze/saving-and-retrieving.ipynb",
-    "tutorials/build-repitition-codes/notebook.ipynb",
-    "tutorials/chsh-inequality/notebook.ipynb",
-    "tutorials/grovers-algorithm/notebook.ipynb",
-    "tutorials/quantum-approximate-optimization-algorithm/notebook.ipynb",
-    "tutorials/repeat-until-success/notebook.ipynb",
-    "tutorials/submitting-transpiled-circuits/notebook.ipynb",
-    "tutorials/variational-quantum-eigensolver/notebook.ipynb",
+NOTEBOOKS_THAT_CANT_BE_MOCKED = [
+    # Notebooks that don't work with the fake backend
+    # (e.g. due to large circuits)
 ]
```

Comment:
Nit because the comment describes the list, not element 0 of the list

```suggestion
# Notebooks that don't work with the fake backend, e.g. due to large circuits
NOTEBOOKS_THAT_CANT_BE_MOCKED = []
```

Also should this have entries in this PR?

Reply:
I think "no" because there aren't any notebooks that "can't" be mocked, they are just broken. I would add them to `NOTEBOOKS_EXCLUDE` but I believe they're all about to be reviewed and edited.

I added this empty list in anticipation of the 100-qubit job in #1158.